haskell-language-server.cabal

```diff
@@ -18,7 +18,7 @@ extra-source-files:
 
 flag agpl
   Description: Enable AGPL dependencies
-  Default:     True
+  Default:     False
```

Comment:
This seems like an unrelated change. It will be confusing for users if we flip this, based on the preferences of the person making a PR.

Reply:
Oh sorry, I didn't mean to include that change. Will remove